ui/helpers/utils/optimism/fetchEstimatedL1Fee.js

```diff
@@ -19,8 +19,10 @@ const OPTIMISM_GAS_PRICE_ORACLE_ABI = [
 const OPTIMISM_GAS_PRICE_ORACLE_ADDRESS =
   '0x420000000000000000000000000000000000000F';
 
-export default async function fetchEstimatedL1Fee(txMeta) {
-  const provider = new Web3Provider(global.ethereumProvider, 10);
+export default async function fetchEstimatedL1Fee(txMeta, ethersProvider) {
```

Comment:
did you want to rename this `ethereumProvider`?

Reply:
I wanted at first, but then I saw we call it ethersProvider in many places, so I wanted to be consistent. Example: `this.ethersProvider = new ethers.providers.Web3Provider(opts.provider);`